tox.ini

```diff
@@ -90,6 +90,8 @@ deps =
 # TODO: force unreleased opentracing v2.0.0 for now
 # git+https://github.com/opentracing/opentracing-python.git@v2.0.0
     git+https://github.com/carlosalberto/opentracing-python.git@scope_managers_integration
+# TODO: this for some reason is needed for an opentracer testcase
```

Comment:
Why? I'd say to check this. It means we're having a side-effect not expected here. Would be great if you can find the reason and update this thread.